### PR TITLE
update packages dependant on http v1

### PR DESCRIPTION
**Pull request by campeis**
Updated reqwest, hyper and wire mock to work with http v1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+- Updated version ot 'http' to '1.1.0'
+- Updated version ot 'hyper' to '1.2.0'
+- Updated version ot 'wiremock' to '0.6.0'
+- Updated version ot 'wiremock' to '0.12.0'
```
Comment (LukeMathWalker) [inline CHANGELOG.md:11]:
```suggestion
- Updated version of 'http' to '1.1.0'
- Updated version of 'hyper' to '1.2.0'
- Updated version of 'wiremock' to '0.6.0'
- Updated version of 'reqwest' to '0.12.0'
```

**Comment by charliermarsh** [comment]
@LukeMathWalker - Is there anything I can do to help support this upgrade?

**Comment by LukeMathWalker** [comment]
I no longer have commit access to this repository (nor publish access to the package on crates.io).
We need one of the maintainers to pick it up (@eopb?).

**Comment by LukeMathWalker** [comment]
Btw, you should also update the README @campeis

File: reqwest-middleware/Cargo.toml [hunk 2]
```diff
@@ -13,14 +13,15 @@ readme = "../README.md"
...
-http = "0.2.0"
-reqwest = { version = "0.11.10", default-features = false, features = ["json", "multipart"] }
+http = "1.1.0"
+reqwest = { version = "0.12.0", default-features = false, features = ["json", "multipart"] }
 serde = "1.0.106"
 task-local-extensions = "0.1.4"
 thiserror = "1.0.21"
 
 [dev-dependencies]
+reqwest = { version = "0.12.0", default-features = false, features = ["json", "multipart", "native-tls"] }
```
Comment (campeis) [inline reqwest-middleware/Cargo.toml:23]: I needed to enable "native-tls" for tests to make the test in README file to be able to connect over https

File: reqwest-tracing/README.md [hunk 3]
```diff
@@ -17,10 +17,10 @@ Attach `TracingMiddleware` to your client to automatically trace HTTP requests:
...
 [dependencies]
 opentelemetry = "0.18"
-reqwest = "0.11"
-reqwest-middleware = "0.1.1"
-reqwest-retry = "0.1.1"
-reqwest-tracing = { version = "0.3.1", features = ["opentelemetry_0_18"] }
+reqwest = "0.12"
+reqwest-middleware = "0.2.5"
+reqwest-retry = "0.4.0"
+reqwest-tracing = { version = "0.4.8", features = ["opentelemetry_0_18"] }
```
Comment (campeis) [inline reqwest-tracing/README.md:23]: these versions should be updated for the next release

**Comment by eopb** [comment]
Thank you for this PR @campeis. I'm sorry it took so long to review.

These changes were also made in https://github.com/TrueLayer/reqwest-middleware/pull/135 so I'm going to close this PR. Thanks again for making these changes.

**Pull request closed without merging**